AppServiceSharedNotepad/CS/README.md

```diff
@@ -0,0 +1,322 @@
+# Using an app service with multiple clients
+We’ll create a simple notepad [app service](https://docs.microsoft.com/en-us/windows/uwp/launch-resume/how-to-create-and-consume-an-app-service).  We'll also create two simple app service clients. One that writes to the notepad service and one that reads from the notepad service.
+
+## Load the projects in Visual Studio
+___
+
+* You can find the source code for this sample by downloading a zip of all of our samples in WIndows 10 IoT Enterprise folder and navigating to the `samples-develop\AppServiceBlinky`. 
+* Make a copy of the folder on your disk and open the projects from Visual Studio. 
+* NotepadService.sln implements the app service and must be started first.
+* NotepadClientBackgroundApplication.sln implements a client that writes messages to the app service.
+* NotepadServiceClientApp.sln implements a windows that retrieves messages from the client and displays them.
+* Make a copy of the folder on your disk and open the projects from Visual Studio. 
+* NotepadService.sln implements the app service and must be started first. 
+* NotepadClientBackgroundApplication.sln implements the app service client.
+
+## Deploy your app
+
+* * *
+
+With the application open in Visual Studio 2019, set the architecture in the toolbar dropdown. If you’re building for UpBaord, select `x64`.
+
+When everything is set up, you should be able to press F5 from each instance of Visual Studio. 
+
+The NotepadService app will deploy and start on the Windows IoT device, and you should see the package family name printed the debug output window on Visual Studio.  
+
+Before pressing F5 for NotepadClientBackgroundApplication and NotepadServiceClientApp projects verify that the value of connection.PackageFamilyName in StartupTask.cs and MainPage.xaml.cs matches the value output in the output window by NotepadService.
+
+If it doesnot any value output, You will get the value by marking the breakpoint in debug mode for 

```

Comment:
"If it doesnot any value output, You will get", does not make sense.   Should it be "If it does not have any output value, you will get" ?

Reply:
Updated it